src/app/item-popup/Plug.tsx

```diff
@@ -44,30 +47,126 @@ export default function Plug({
 
   const tooltip = () => <DimPlugTooltip item={item} plug={plug} wishlistRoll={wishlistRoll} />;
 
-  // Is this the currently active plug - either because it's what's slotted in game or the user has clicked to preview it
-  const plugged = plug === socketInfo.plugged;
   const selectable = socketInfo.plugOptions.length > 1;
 
   return (
     <div
       key={plug.plugDef.hash}
       className={clsx('socket-container', {
         disabled: !plug.enabled,
-        notChosen: !plugged,
         selectable,
-        // This has been selected by the user but isn't the original plugged item
-        selected: socketInfo.actuallyPlugged && plugged,
-        // Another plug was selected by the user
-        notSelected: socketInfo.actuallyPlugged && !plugged && plug === socketInfo.actuallyPlugged,
         notIntrinsic: !itemCategories.includes(ItemCategoryHashes.WeaponModsIntrinsic),
-        cannotRoll: plug.cannotCurrentlyRoll,
       })}
       onClick={hasMenu || selectable ? doClick : undefined}
     >
-      <PressTip tooltip={tooltip}>{contents}</PressTip>
+      {socketInfo.isReusable ? (
+        <PerkCircleWithTooltip
+          item={item}
+          plug={plug}
+          wishlistRoll={wishlistRoll}
+          socketInfo={socketInfo}
+        />
+      ) : (
+        <>
+          <PressTip tooltip={tooltip}>{contents}</PressTip>
+          {wishlistRoll?.wishListPerks.has(plug.plugDef.hash) && (
+            <AppIcon
+              className="thumbs-up"
+              icon={thumbsUpIcon}
+              title={t('WishListRoll.BestRatedTip')}
+            />
+          )}
+        </>
+      )}
+    </div>
+  );
+}
+
+/**
+ * a perk circle and its associated thumbs up or lack thereof.
+ * this belongs inside an element with a css position, so thumbs up can position itself.
+ */
+export function PerkCircleWithTooltip({
+  item,
+  plug,
+  socketInfo,
+  wishlistRoll,
+}: {
+  item: DimItem;
+  plug: DimPlug;
+  socketInfo: DimSocket;
+  wishlistRoll?: InventoryWishListRoll;
+}) {
+  const plugged = plug === socketInfo.plugged;
+
+  // Another plug was selected by the user
+  const notSelected = socketInfo.actuallyPlugged && !plugged && plug === socketInfo.actuallyPlugged;
+  // This has been selected by the user but isn't the original plugged item
+  const selected = socketInfo.actuallyPlugged && plugged;
+  const cannotRoll = plug.cannotCurrentlyRoll;
+
+  const tooltip = () => <DimPlugTooltip item={item} plug={plug} wishlistRoll={wishlistRoll} />;
+  return (
+    <>
+      <PressTip tooltip={tooltip}>
+        <PerkCircle plug={plug} {...{ plugged, notSelected, selected, cannotRoll }} />
```

Comment:
I generally avoid spread props, there's a performance cost that I'm not quite sure of, and they don't quite work with types the way I'd hope (an extra prop spread into the component won't be flagged if the component no longer uses them).